cmd/generate/cluster/template/command.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	name        = "{{ .Cluster }}"
-	description = "Conformance tests for cluster {{ .Cluster }}."
+	description = "Conformance tests for cluster scope {{ .Cluster }}."
 )
 
```

Comment:
Boyscouting: I find it important to get the terminology straight. When we call it "cluster scope", we should do it most consistently everywhere.